THIRD-PARTY-NOTICES.TXT

```diff
@@ -57,3 +57,20 @@ AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
 LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
 OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
 THE SOFTWARE.
+
+License notice for OpenTelemetry .NET (https://github.com/open-telemetry/opentelemetry-dotnet)
+----------------------------------------------------------------------------------------------
+
+Copyright 2023 OpenTelemetry Authors
```

Comment:
This is what OpenTelemetry uses https://github.com/open-telemetry/opentelemetry-dotnet/blob/af57ae87871aac7e201a03255ac8c81e5747bc07/src/OpenTelemetry.Api/Baggage.cs#L1-L2

```suggestion
Copyright The OpenTelemetry Authors
```

Reply:
Is it worth filing an issue in Azure SDK? I took it from there: https://github.com/Azure/azure-sdk-for-net/blob/main/NOTICE.txt#L232-L235

Reply:
> Is it worth filing an issue in Azure SDK? I took it from there: https://github.com/Azure/azure-sdk-for-net/blob/main/NOTICE.txt#L232-L235

@rajkumar-rangaraj could you follow up and handle this?

Reply:
I will get this handled in Azure SDK. 
When we earlier vendored in Azure SDK, OpenTelemetry had that license format.

It's worth to update the Readme to check on the license formats too.